PyDMXControl/_Colors.py

```diff
@@ -62,11 +123,47 @@ def to_dict(colors: List[int]) -> Dict[str, int]:
 
     @staticmethod
     def to_tuples(colors: List[int]) -> List[Tuple[str, int]]:
-        """ Assumes RGBWA """
+        """Convert a color from a list to a list of tuples.
+        
+        Assumes RGBWA.
+        
+        Parameters
+        ----------
+        colors: Color to convert to tuples.
+
+        Returns
+        -------
+        List[Tuple[str, int]]
+            Color as a list of tuples.
+
+        Examples
+        --------
+
+        >>> PyDMXControl.Colors.to_tuples([1,2,3,4,5]) 
+        [('R', 1), ('G', 2), ('B', 3), ('W', 4), ('A', 5)]
+        """
         return [(k, v) for k, v in Colors.to_dict(colors).items()]
 
     @staticmethod
     def to_hex(colors: List[int]) -> str:
+        """Convert a color from list to hex form.
+
+        Parameters
+        ----------
+        colors: Color to convert to tuples.
+
+        Returns
+        -------
+        str
+            Color as a hex string.
+
+        Examples
+        --------
+
+        >>> PyDMXControl.Colors.to_hex([95, 93, 12, 18, 128])
```

Comment:
```suggestion
        >>> Colors.to_hex([95, 93, 12, 18, 128])
```
Remove package name.